### DISCUSSION TITLE
Quarkus Amazon Clients and resilience

### DISCUSSION THREAD

**Discussion by tor-jorgen-yara** [quarkusio/quarkus#22454] [Q&A] [answered]
The Amazon SDKs implement automatic retry logic, but how should resilience be handled from the Quarkus Amazon Clients?
For the asynchronous DynamoDB client, the only things I can see that can be configured is: 
- `quarkus.dynamodb.async-client.xxx-timeout`

Is the underlying functionality in the SDKs turned off, so that we can use `smallrye-fault-tolerance` for this, or how should this be handled correctly?

**Comment by gsmet** [comment, marked as answer]
I think it's just that it hasn't been implemented.

The Amazon Services extensions have been extracted here https://github.com/quarkiverse/quarkus-amazon-services for easier contribution so feel free to contribute there.

**Reply by tor-jorgen-yara** [reply]
Thanks for replying Guillaume. I had a look at the code and the default AWS retry policy is intact. I'll create a PR if we need to tweak the default policy...
